dotCMS/src/main/java/com/dotcms/util/I18NUtil.java

```diff
@@ -1,5 +1,7 @@
 package com.dotcms.util;
 
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
 import com.liferay.portal.language.LanguageException;
 import com.liferay.portal.language.LanguageUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.liferay.portal.language.LanguageUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)